pkg/workloads/kubernetes.go

```diff
@@ -69,5 +88,55 @@ func fetchK8sLabels(containerLbls map[string]string) (map[string]string, error)
 		delete(k8sLabels, k8sConst.PolicyLabelServiceAccount)
 	}
 
+	// If the pod has been injected with an Istio sidecar proxy compatible with
+	// Cilium, add a label to notify that.
+	// If the pod already contains that label to explicitly enable or disable
+	// the sidecar proxy mode, keep it as is.
+	if _, ok := k8sLabels[k8sConst.PolicyLabelIstioSidecarProxy]; !ok &&
```

Comment:
Should this check for "false" besides non-existing label as well?

Reply:
Right. Initially I defined this label as having no value. I didn't update this test.

Reply:
Oh, actually this is the behavior I intended. If the user knows better than Cilium can guess, and sets the label to either `"false"` or `"true"`, we don't overwrite that value.

Reply:
👍 makes sense. Just needs to be documented then.